cooperator/models/company.py

```diff
@@ -100,21 +100,61 @@ def _compute_base_logo(self):
         translate=True,
         help="Text to display aside the checkbox to approve the generic rules.",
     )
+    cooperator_certificate_mail_template = fields.Many2one(
+        comodel_name="mail.template",
+        string="Certificate email template",
+        domain="[('model', '=', 'res.partner')]",
```

Comment:
an actual domain or a string are allowed for this attribute. is there a particular reason to use the string version?

Reply:
I copied this from `stock` ;) No particular reason.